src/pages/changePassword/index.ts

```diff
@@ -0,0 +1,84 @@
+import ProfileLayout from "../../layouts/profile";
+
+import template from "./template.hbs";
+import Component from "../../utils/Component";
+import * as styles from "./styles.module.scss";
+import { getFormData, isDisableForm, setErrors } from "../../utils/validation";
+import { changePasswordContentProps } from "./changePassword";
+
+
+const errors = {
+  oldPassword: '',
+  newPassword: '', 
+  repeatPassword: '', 
+}
+
+const values = {
+  oldPassword: '',
+  newPassword: '', 
+  repeatPassword: '', 
+}
+
+let isDisabledForm = false;
+
+class ChangePassword extends Component {
```

Comment:
Предлагаю уточнить название, чтобы оно стало исчерпывающим
```suggestion
class ChangePasswordPage extends Component {
```